Sprint 4.

ASevastyanov opened the pull request:
Версия 1.0
Верстка основного дизайна по макету с Figma.
Настройка основной логики приложения на моках.

Самодеятельность:
Добавлено логирование в консоль, для всей логики в приложение. 
Добавлено лоигика для скрытия рамки(Рамка с покраской, для правильного и не правильного ответа) после какого либо ответа и перед началом новой игры.

ASevastyanov committed:
Выполненное задание спринта 4.
Версия 1.0
Верстка основного дизайна по макету с Figma.
Настройка основной логики приложения на моках.

helecta commented on MovieQuiz/Presentation/MovieQuizViewController.swift:
```diff
@@ -1,72 +1,183 @@
+
+// MARK: - Структуры приложения
+
+//Структура для конвертации вопросов с мока
```
Здорово, что добавляешь комментарии. Если добавлять через /// , а не // , то тогда можно будет быстро прочитать документацию, если зажать кнопку option и навести на класс.

ASevastyanov replied:
Исправлено

helecta commented on MovieQuiz/Presentation/MovieQuizViewController.swift:
```diff
@@ -1,72 +1,183 @@
+    private var currentQuestionIndex = 0 //счетчик вопросов
+    private var correctAnswers = 0 //кол-во правильных ответов
+    
+    @IBOutlet private var imageView: UIImageView!
```
IBOutlet обычно weak. Дальше по курсу будет понятно почему так.

ASevastyanov replied:
Пока не совсем понял, исправил, но для чего не разобрался.

helecta commented on MovieQuiz/Presentation/MovieQuizViewController.swift:
```diff
@@ -1,72 +1,183 @@
+        imageView.image = step.image
+        textLabel.text = step.question
+        
+        self.imageView.layer.borderWidth = 0 //Убрать рамку с появление нового вопроса(showAnswerResult)
```
Тут `self` необязательно использовать, поэтому лучше его убрать.
Подробнее можешь почитать в [код стайле](https://github.com/tutu-ru/swift-style-guide#комментарии)

ASevastyanov replied:
Исправил, спасибо!)

ASevastyanov committed:
версия 1.1

Фикс цвета StatusBar (Изменен на белый цвет)
Фикс кнопок (Да, Нет), блокировка в время показа результата (Добавлен новый метод)
Фикс комментариев после ревью
Доработка документации, описание методов и констант
